test/TestProjects/AgrifoodAsExample-TypeSpec/resource.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
This file is printed from API `getArmResources`. It takes all the tsp spec under the same folder as input.